test/swagger.spec.js

```diff
@@ -101,331 +112,257 @@ describe('Swagger docs', function () {
-            const api = apis['GET /people'].operations[0]
+            const api = apis['/people'].get
             const parameters = {}
             _.forEach(api.parameters, function (item) {
                 parameters[item.name] = item
             })
             expect(parameters).to.have.property('include')
-            expect(parameters.include).to.have.property('paramType', 'query')
+            expect(parameters.include).to.have.property('in', 'query')
             expect(parameters.include).to.have.property('type', 'string')
             expect(parameters).to.have.property('fields')
-            expect(parameters.fields).to.have.property('paramType', 'query')
-            expect(parameters.fields).to.have.property('type', 'fields')
+            expect(parameters.fields).to.have.property('in', 'query')
+            expect(parameters.fields).to.have.property('type', 'object')
             expect(parameters).to.have.property('sort')
-            expect(parameters.sort).to.have.property('paramType', 'query')
+            expect(parameters.sort).to.have.property('in', 'query')
             expect(parameters.sort).to.have.property('type', 'string')
             expect(parameters).to.have.property('page')
-            expect(parameters.page).to.have.property('paramType', 'query')
-            expect(parameters.page).to.have.property('type', 'page')
+            expect(parameters.page).to.have.property('in', 'query')
+            expect(parameters.page).to.have.property('type', 'object')
             expect(parameters).to.have.property('filter')
-            expect(parameters.filter).to.have.property('paramType', 'query')
-            expect(parameters.filter).to.have.property('type', 'filter')
+            expect(parameters.filter).to.have.property('in', 'query')
+            expect(parameters.filter).to.have.property('type', 'object')
         })
         it('should describe POST /people parameters', function () {
-            const api = apis['POST /people'].operations[0]
-            expect(api.consumes).to.eql(['application/json'])
+            const api = apis['/people'].post
+            //expect(api.consumes).to.eql(['application/json'])
             const parameters = {}
             _.forEach(api.parameters, function (item) {
                 parameters[item.name] = item
             })
             expect(parameters).to.have.property('body')
-            expect(parameters.body).to.have.property('paramType', 'body')
-            expect(parameters.body).to.have.property('type', 'people')
+            expect(parameters.body).to.have.property('in', 'body')
         })
         it('should describe GET /people/changes/streaming parameters', function () {
-            const api = apis['GET /people/changes/streaming'].operations[0]
-            expect(api.parameters).to.have.length(0)
+            const api = apis['/people/changes/streaming'].get
+            expect(api.parameters).to.not.exist
```

Comment:
Yes we do have a `/changes/streaming` endpoint that requires query parameters. Though it's not tested here. This set of tests was concerned with all routes that were prefixed with `/people/` thus `/people/changes/streaming` was tested. It didn't look at `/pets/` or `/changes/`. We could add another section for `/changes/` to check that `/changes/streaming` exists as expected.